test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -1951,9 +1952,12 @@ public class ModelBindingTest
 
             // Assert
             var result = await ReadValue<UserWithAddress>(response);
+
+            // Though posted content did not contain any Addresses, it is bound as a single-element List containing
```

Comment:
This behavior seems like it's still a bug. @dougbu changes change it to be a little cleaner, but ideally this would be an empty list.